src/main/webapp/user_profile.html

```diff
@@ -115,5 +49,25 @@ <h1>CREATE A NEW TASK: </h1>
             </form>
         </div>
     </div>
+    <div id="editTaskModal">
+        <div id="editModal">
+            <span id="edit-close-button" onclick="closeEditModal()">&times;</span>
+            <form id="edit-task-form" action="/tasks/edit" method="POST">
+                <h1>EDIT THE CURRENT TASK: </h1>
+                <div>
+                    <label for="edit-content-input">Task Detail:</label>
+                    <br/>
+                </div>
+                <textarea name="edit-content-input" id="edit-content-input"></textarea>
+                <br/>
+                <label for="edit-point-input">Rewarding Points: </label>
+                <input type="number" id="edit-point-input" name="edit-input" min="0" max="200">
```

Comment:
"edit-input" seems like a generic name for this part of the form; how about "edit-point-input", to match the ID?

Reply:
I change the name to "reward-input" to match the name of the create task form in the new commit.